NodePropertyPalette/BooleanToTextConverter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Data;
+
+namespace NodePropertyPalette
+{
+    /// <summary>
+    /// Convert a bool to a Yes/No string.
+    /// </summary>
+    public class BooleanToTextConverter : IValueConverter
```

Comment:
Hi @mmisol We have a lot of converters defined in Dynamo, can you explore if it would be possible to use the ones defined in Dynamo?

Reply:
I took a quick look and didn't find any that converted from boolean to yes/no

Reply:
I guess I could use `EnumToBooleanConverter` if I defined an auxiliary enum with Yes/No values. Do you think that's a better idea?

Reply:
@mmisol Since we will be setting up examples for 3rd party developers, I feel sticking with coverters Dynamo SDK can provide will be a bit better in this case, we can also add needed converter into DynamoCore, but for now would you stick to whatever we have? Appreciate it